electron-builder.yml

```diff
@@ -32,11 +32,6 @@ win:
       arch:
         - x64
         - arm64
-  signtoolOptions:
-    sign: "./scripts/sign-win.js"
-    signingHashAlgorithms:
-      - sha256
-    publisherName: "ZOO CORPORATION" # needs to be exactly like on Digicert
```

Comment:
Now setting those in the script, as it would fail on dev builds otherwise, looking for a cert :'(